src/main/java/org/folio/circulation/domain/validation/LoanPolicyValidator.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.domain.representations.CheckOutByBarcodeRequest.ITEM_BARCODE;
+import static org.folio.circulation.resources.RenewalValidator.loanPolicyValidationError;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+import static org.folio.circulation.support.results.Result.failed;
+import static org.folio.circulation.support.results.Result.succeeded;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.policy.LoanPolicy;
+import org.folio.circulation.support.results.Result;
+
+public class LoanPolicyValidator {
+
```

Comment:
Removed it. Actually, I've just realized that this validation is done twice. It is a copy of `RegularCheckOutStrategy.refuseWhenItemIsNotLoanable()` which was a part of the `CheckOutStrategy.checkOut()` process. I needed to perform this validation earlier in the chain to accumulate it together with other validation errors and return a failure before the actual check-out, this is why I've extracted it to a separate class. As a result, it's now done twice - first time directly in the check-out chain (it's skipped in case of a OverrideCheckOutStrategy) and second time in the strategy. So I've removed this second validation.